main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
Is this POC scoped to something like a "go-restful-poc" directory at the root of the project? It should be encapsulated.

Reply:
Reorganizing to accomplish this as well